Sources/TokamakDemo/TokamakDemo.swift

```diff
@@ -21,45 +21,97 @@ import SwiftUI
 import TokamakDOM
 #endif
 
+func title<V>(_ view: V, title: String) -> AnyView where V: View {
+  if #available(OSX 10.16, iOS 14.0, *) {
+    return AnyView(view.navigationTitle(title))
+  } else {
+    #if !os(macOS)
+    return AnyView(view.navigationBarTitle(title))
+    #else
+    return AnyView(view)
+    #endif
+  }
```

Comment:
This should fix the build error on CI:

```suggestion
  #if !os(WASI)
  if #available(OSX 10.16, iOS 14.0, *) {
    return AnyView(view.navigationTitle(title))
  }
  #elseif !os(macOS)
  return AnyView(view.navigationBarTitle(title))
  #else
  return AnyView(view)
  #endif
```

Reply:
That doesn’t work — it errors out with “Missing return in a function expected to return 'AnyView'”

Reply:
I'm sorry, I didn't test the native bit, here's what did work then:

```suggestion
  #if !os(WASI)
  if #available(OSX 10.16, iOS 14.0, *) {
    return AnyView(view.navigationTitle(title))
  } else {
    return AnyView(view)
  }
  #elseif !os(macOS)
  return AnyView(view.navigationBarTitle(title))
  #else
  return AnyView(view)
  #endif
```

Reply:
I ended up just stubbing out the `navigationTitle` modifier too 🤷